vm/src/builtins.rs

```diff
@@ -74,7 +74,21 @@ fn builtin_any(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
 }
 
 // builtin_ascii
-// builtin_bin
+
+fn builtin_bin(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
+    arg_check!(vm, args, required = [(number, Some(vm.ctx.int_type()))]);
+
+    let n = match number.borrow().kind {
```

Comment:
There is a helper function you could use here: `let n = objint::getvalue(number);`.

Reply:
Done. Thanks! :)